CITIES.md

```diff
@@ -2338,4 +2338,5 @@ The municipality id (IBGE code) can be found on [Wikipedia](https://pt.wikipedia
 | 2328 | Vera Cruz - BA | :white_check_mark: | | [PR](https://github.com/okfn-brasil/querido-diario/pull/354) |
 | 2329 | Wenceslau Guimarães - BA | :white_check_mark: | | [PR](https://github.com/okfn-brasil/querido-diario/pull/354) |
 | 2330 | Xique-Xique - BA | :white_check_mark: | | [PR](https://github.com/okfn-brasil/querido-diario/pull/354) |
-| 2331 | Pau dos Ferros - RN | :white_check_mark: | [issue](https://github.com/okfn-brasil/querido-diario/issues/489) | [PR](https://github.com/okfn-brasil/querido-diario/pull/494) |
\ No newline at end of file
+| 2331 | Adolfo - SP | :white_check_mark: | [issue](https://github.com/okfn-brasil/querido-diario/issues/461) | [PR](https://github.com/okfn-brasil/querido-diario/pull/475)
```

Comment:
Ops, precisa estar em ordem alfabética pra facilitar. :)

Reply:
Mas o arquivo em si já não está seguindo essa ordem, em grande parte pelo menos. Além de se eu for colocar essa cidade lá em cima, significa que teria que ser alterado os id das mais de duas mil cidades subsequentes?
Concordo que em ordem alfabética é bem melhor, mas não deveriamos desconsiderar o uso do id então?

Poderíamos criar seções `Capitais`, `A`, `B` ...  Ou por estado, e dentro do estado capital primeiro e depois ordem alfabética. O que pensa sobre?

Reply:
Putz, tem razão! Pode deixar assim mesmo. Obrigada!